config/configuration.rb

```diff
@@ -19,5 +24,21 @@ def app_sharing_enabled?
       @app_sharing_enabled = ENV['OOD_APP_SHARING'].present?
     end
     alias_method :app_sharing_enabled, :app_sharing_enabled?
+
+    def custom_brand_bg_color
+      return @custom_brand_bg_color if defined? @custom_brand_bg_color
+      @custom_brand_bg_color = ENV.values_at('OOD_BRAND_BG_COLOR', 'BOOTSTRAP_NAVBAR_DEFAULT_BG', 'BOOTSTRAP_NAVBAR_INVERSE_BG').compact.first
+    end
+
+    def custom_brand_link_active_bg_color
+      return @custom_brand_link_active_bg_color if defined? @custom_brand_link_active_bg_color
```

Comment:
Same as above, not sure we need to cache this. And possibly renaming this to `brand_link_active_bg_color`.

Reply:
Done: 72c57d3